homework/Vasili_Zasinets/Shape/src/main/java/com/tutrit/java/quickstart/shape/fabric/ShapeProvider.java

```diff
@@ -0,0 +1,11 @@
+package com.tutrit.java.quickstart.shape.fabric;
+
+import com.tutrit.java.quickstart.shape.AbstractShape;
+
+public class ShapeProvider implements Provider{
+
+    @Override
+    public AbstractShape getShape(String name, double v) {
+        return null;
```

Comment:
Return null is very dangerous. May happen NullPointException.
It is hard to debug.